docs/components/image/ThemedImageFigure.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { ImageFigureProps } from "./ImageFigure";
+import { ThemedImage, ThemedImageProps } from "./ThemedImage";
+import cn from "classnames";
+export type ThemedImageFigureProps = Omit<ImageFigureProps, "src"> &
+  ThemedImageProps;
+
+export function ThemedImageFigure(
+  props: ThemedImageFigureProps
+): React.ReactNode {
+  const {
+    caption,
+    margin = 40,
+    captionSpacing = null,
+    shadow = false,
+    borderRadius = false,
+    ...rest
+  } = props;
+
+  return (
+    <figure
+      className="block -mx-4 text-center sm:-mx-4 md:-mx-7 lg:-mx-12"
+      style={{ marginTop: `${margin}px`, marginBottom: `${margin}px` }}
+    >
+      <div
+        className={cn(
+          "relative inline-block max-w-full overflow-hidden border-box",
+          {
+            "rounded-md": borderRadius,
+          }
+        )}
+        style={{ fontSize: 0 }}
```

Comment:
Nitpick:

Could use `text-[0px]` in the regular classname unless you need this here for specificity